config.go

```diff
@@ -9,6 +9,7 @@ import (
 type Config struct {
 	RequiredAlias map[string]string
 	Rules         []*Rule
+	StrictMode    bool
```

Comment:
Im wondering if there's a more specific flag/config name we can use here (I know `strict` is what the todo said, but before exposing it in the "api" it might be nice to see if there's something more descriptive we can use). 

WDYT about something like `DisallowUnaliased`, with flag `--no-unaliased` and description `do not allow unaliased imports of aliased packages`?

(Thanks again for the PR!)

Reply:
@julz I think it's nice idea!
I'll fix this.

Reply:
I've fixed this https://github.com/julz/importas/pull/4/commits/25f1bb8bc1910827ff0e50f698033efcaba6e398